pages/docs/tutorials/getting-started/request_reply.md

```diff
@@ -0,0 +1,240 @@
+---
```

Comment:
FYI, you used an underscore in your file name `request_reply.md`, but **please update your file name to use a dash instead: `request-reply.md`** to follow our convention in AsyncAPI docs. ✌🏽

Reply:
👍